### PR TITLE
Adds Unlink Wallet Flow

### PR DESCRIPTION
**Pull request by nitaliano**
**Description**

Adds Unlink Wallet Flow

Screenshot 2024-03-13 at 11 53 49 AM
Screenshot 2024-03-13 at 11 53 40 AM

### CODE REVIEW

File: apps/dapp-console/app/settings/components/LinkedWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    () => setIsDialogOpen(false),
+    [setIsDialogOpen],
+  )
+
+  return (
+    <div className="flex flex-row gap-2">
+      <Input className="rounded" value={wallet.address} disabled />
+      <Dialog open={isDialogOpen} onOpenChange={setIsDialogOpen}>
+        <DialogTrigger asChild>
+          <Button className="rounded" variant="secondary" size="icon">
```
Comment (tarunkhasnavis) [inline apps/dapp-console/app/settings/components/LinkedWallet.tsx:33]: q: do we need `aria-label` when button text is not clear (maybe it is clear inside <RiCloseLine>)

File: apps/dapp-console/app/settings/components/LinkedWallet.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+              Are you sure you want to unlink this wallet?
+            </Text>
+            <Text
+              as="p"
+              className="cursor-default mb-4 text-muted-foreground text-base"
+            >
+              {shortenAddress(wallet.address)}
+            </Text>
+
+            <Button className="rounded w-full mb-2" onClick={handleUnlink}>
```
Comment (tarunkhasnavis) [inline apps/dapp-console/app/settings/components/LinkedWallet.tsx:53]: Could be cool in the future to show toasts for linking and unlinking

**Review by tarunkhasnavis — APPROVED** [review]
lgtm! 

Whenever we add new packages should we be pushing the `pnpm-lock.yaml` file up?

File: apps/dapp-console/app/settings/components/LinkedWallet.tsx (hunk 1 above)
Comment (nitaliano) [reply apps/dapp-console/app/settings/components/LinkedWallet.tsx:33]: yea i think semantically you do want to use an `aria-label` when a button just contains svg, will add!

**Pull request merged**